src/utils/random-bigint.ts

```diff
@@ -0,0 +1,5 @@
+import { randomNumber, randomNumberBetween } from '@makerx/ts-dossier'
```

Comment:
I'm assuming this is only used in tests? Probably makes sense to move to src/tests/utils